pyhap/hap_protocol.py

```diff
@@ -246,27 +246,33 @@ def _process_one_event(self) -> bool:
         logger.debug(
             "%s (%s): h11 Event: %s", self.peername, self.handler.client_uuid, event
         )
-        if event in (h11.NEED_DATA, h11.ConnectionClosed):
```

Comment:
This bug was missed because there was no explicit test and it looked like it was covered because we did match on NEED_DATA